packages/snaps-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [14.2.2]
+
+### Fixed
+
+- Throw clearer error when the NPM registry returns 404 ([#3602](https://github.com/MetaMask/snaps/pull/3602))
+- Only log error that originally caused the crash ([#3569](https://github.com/MetaMask/snaps/pull/3569))
```

Comment:
Nit: The changelog could be clearer, it’s hard to understand what’s being fixed without checking the PR.